### DISCUSSION TITLE
Optimize for Apple M2/M3 chips to slice faster?

### DISCUSSION THREAD

**Discussion by tll007** [OrcaSlicer/OrcaSlicer#2987] [General]
is there any way to optimize the slicer for better performance... particularly for the M2/M3 chips for Apple?
I have an M2 Max and watching the slicing calculate, I can't help but wonder if it could go faster if it could be optimized.

**Comment by ExR90** [comment]
I have a M3 Max and it's painfully slow. Checking Activity Monitor and it seems like the computer isn't really breaking a sweat. Not sure if it's a threading issue or not being GPU optimized or what. Thankfully I don't slice big/complex files that frequently, but it is still noticeable either way. Now if I could only get Fusion dev's to do the same........lol

**Comment by ajaytanna** [comment]
HI, any of you notice nightly builds crowding the application folder with tutk_log*****.txt files - 3 of each on every launch of the app. Any solution to this?
